js/styles/inlineStyle.js

```diff
@@ -11,6 +11,24 @@ module.exports = {
     }
   },
 
+  iPadVerticalVideoStyle: {
```

Comment:
I would think that the 4 bottom can be put on CSS. While width and height can be assigned on the fly in controller.js